server/odc-service/src/main/java/com/oceanbase/odc/service/sqlplan/model/SqlPlanTaskResult.java

```diff
@@ -20,13 +20,40 @@
 @Data
 public class SqlPlanTaskResult {
 
-    private Long totalStatements;
+    private long totalStatements = 0;
 
-    private Long finishedStatements;
+    private long finishedStatements = 0;
 
-    private Long successStatements;
+    private long succeedStatements = 0;
 
-    private Long failedStatements;
+    private long failedStatements = 0;
+
+    /**
+     * sql execution json file download url
+     */
+    private String sqlExecuteJsonFileDownloadUrl;
+
+    /**
+     * DQL result set download url
+     */
+    private String csvResultSetZipDownloadUrl;
+
+    /**
+     * error record download url
+     */
+    private String errorRecordsFileDownloadUrl = null;
+
+
+    public void incrementFinishedStatements() {
```

Comment:
may set incrementFinishedStatements as private method,
referenced in other 2 methods.

may simplify to successStatement(), failStatement()

Reply:
Have been modified.